src/pages/manager/OpenAccountPage.js

```
@@ -1,13 +1,23 @@
 import { expect } from '@playwright/test';
 
-export class AddCustomerPage {
+export class OpenAccountPage {
   constructor(page) {
     this.page = page;
+    this.selectCurrencyDropdown = page.locator('#currency');
   }
 
   async open() {
     await this.page.goto(
       '/angularJs-protractor/BankingProject/#/manager/openAccount',
     );
   }
+  
+ async selectCurrency(currency) {
+    await this.selectCurrencyDropdown.selectOption(currency);
```

Add an explicit assertion after selecting the currency to make the expectation clear and reduce silent failures. For example:

```javascript
await this.selectCurrencyDropdown.selectOption({ label: currency });
await expect(this.selectCurrencyDropdown).toHaveValue(expectedValue);
```

This makes it obvious whether the select actually changed to the expected option.